dev-tools/mage/settings.go

```diff
@@ -310,6 +317,20 @@ var (
 	beatVersionOnce  sync.Once
 )
 
+// BeatQualifiedVersion returns the Beat's qualified version.  The value can be overwritten by
+// setting BEAT_VERSION_QUALIFIER in the environment.
```

Comment:
Think we should put `BEAT_VERSION_QUALIFIER` in the godocs for the `release` targets (duplicated in each magefile)?

```
filebeat akroh$ mage -h package
mage package:

Package packages the Beat for distribution.
Use SNAPSHOT=true to build snapshots.
Use PLATFORMS to control the target platforms.
```

Reply:
Great idea, I'll add that now.